hardhat.config.ts

```diff
@@ -12,7 +12,6 @@ import 'hardhat-gas-reporter'
 import 'hardhat-contract-sizer'
 import 'hardhat-abi-exporter'
 import 'solidity-coverage'
-import '@nomiclabs/hardhat-etherscan'
```

Comment:
@JhChoy `import '@openzeppelin/hardhat-upgrades'` Why this one is missing?

Reply:
Why is that needed?

Reply:
https://docs.openzeppelin.com/upgrades-plugins/1.x/api-hardhat-upgrades#verify @JhChoy  It helps automatically verification in etherscan

Reply:
What kinds of help? The contract verification command works now.
We are not using
```
npx hardhat verify --network mainnet PROXY_ADDRESS
```
We use
```
npx hardhat --network 1101 etherscan-verify
```
as I mentioned in the description of this PR.